fontbe/src/features.rs

```diff
@@ -143,19 +143,63 @@ impl<'a> FeaVariationInfo<'a> {
                 let axis = self.axes.get(tag).unwrap();
                 let pos = match pos {
                     AxisLocation::User(coord) => {
-                        UserCoord::new(coord.to_f32()).to_normalized(&axis.converter)
+                        UserCoord::new(coord.0).to_normalized(&axis.converter)
                     }
                     AxisLocation::Design(coord) => {
-                        DesignCoord::new(coord.to_f32()).to_normalized(&axis.converter)
+                        DesignCoord::new(coord.0).to_normalized(&axis.converter)
                     }
-                    AxisLocation::Normalized(coord) => NormalizedCoord::new(coord.to_f32()),
+                    AxisLocation::Normalized(coord) => NormalizedCoord::new(coord.0),
                 };
                 (*tag, pos)
             })
             .collect()
     }
 }
 
+#[derive(Debug)]
+struct UnsupportedLocationError(NormalizedLocation);
+
+impl UnsupportedLocationError {
+    fn new(loc: NormalizedLocation) -> UnsupportedLocationError {
+        UnsupportedLocationError(loc)
+    }
+}
+
+impl std::error::Error for UnsupportedLocationError {
+    fn source(&self) -> Option<&(dyn std::error::Error + 'static)> {
+        None
+    }
+}
+
+impl Display for UnsupportedLocationError {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        f.write_fmt(format_args!("No variation model for {:?}", self.0))?;
+        Ok(())
+    }
+}
+
+#[derive(Debug)]
+struct MissingTentError(Tag);
```

Comment:
if these two errors are always returned in the same place (like are two error conditions of the same logic) then maybe an enum makes sense 🤷

Reply:
Maaybe ... how about we hold that idea until we add a third